src/plugins/kibana/public/visualize/editor/agg_group.js

```diff
@@ -40,6 +42,29 @@ uiModules
           if (count < schema.max) return true;
         });
       });
+
+      const drake = dragula({
+        containers: $('.vis-editor-agg-group', $el).toArray(),
```

Comment:
mind using custom attributes for this selector (and the others in this file)? The amount of css selectors here is unsettling me a bit